README.md

```diff
@@ -1,3 +1,124 @@
 # cloud-sql-go-connector
 *Warning*: This project is experimental, and is not an officially supported 
 Google product.
+
+The _Cloud SQL Go Connector_ provides IAM authorization to an app's connections 
+to a Cloud SQL instance. It provides connection level authorization only; it 
+_does not_ provide a network path to an instance that doesn't already exist 
+(ie. you will still be unable to connect via Private IP without access to the 
+correct VPC). For more information, see 
+[About the Cloud SQL Auth proxy][about-proxy].
+
+[about-proxy]: https://cloud.google.com/sql/docs/mysql/sql-proxy
+
+The _Cloud SQL Go Connector_ is an experimental new version of the 
+[Cloud SQL proxy dialer](). It's API is considered unstable and may change in 
+the future. Please use at your own risk. 
+
+[proxy-dialer]: https://github.com/GoogleCloudPlatform/cloudsql-proxy/tree/main/proxy#cloud-sql-proxy-dialer-for-go
+
+## Installation
+
+First, clone this repo into a folder relative to your project:
```

Comment:
Should there be a short comment like, "Until this module is published, we will manually install it using a replace directive"?

Reply:
I'm okay not explicitly saying this. I think it's probably obvious from the non standard approach.